src/i18n/fr/strings.txt

```diff
@@ -80,3 +81,8 @@ wizard_change_server=Autre serveur
 wizard_login_button=Connexion
 wizard_next_button=Suivant
 wizard_skip_button=Sauter
+site=Site
+change=Change
+save=Save
+default_journal=Default journal
+try_again_question=Try again?
```

Comment:
settings and try again haven't been used anywhere in application.... removing it from all lang strings